tests/conftest.py

```diff
@@ -36,30 +36,22 @@ def event_loop():
 
 
 @pytest.fixture(scope="session")
-async def bot_initialize():
+async def bot():
     """Called at the beginning of the testing session.
-    Modifies the configuration of the bot to make it suited for testing
+    Starts the bot with the testing setting in another thread
 
     Yields:
         None: wait for the testing session to end
     """
-    config_map['token'] = config_map['test']['token']
-    config_map['meme']['n_votes'] = 1
     warnings.filterwarnings("ignore",
                             message="If 'per_message=False', 'CallbackQueryHandler' will not be tracked for every message.")
-    yield None
-
-
-@pytest.fixture(scope="session")
-async def bot(bot_initialize):
-    """Called at the beginning of the testing session.
-    Starts the bot with the testing setting in another thread
+    for test_key in config_map['test']:
+        if test_key in config_map:
+            config_map[test_key] = config_map['test'][test_key]
+        if test_key in config_map['meme']:
```

Comment:
should be `elif`?
can happen that both if are triggered in the same execution?

Reply:
Nope, they can and will probably happen both